### DISCUSSION TITLE
arrow seems to depend on some gnulib headers not present

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#7598] [General]
fails with missing execinfo.h so i temporarily copied over the gnuwin32 execinfo.h and features.h to mingw*/include and it builds no problem, but leaving those two includes might cause problems with other packages down the line i suspect.

**Comment by Biswa96** [comment]
How can one reproduce your issue? Please provide exact steps.

**Reply by revelator** [reply]
late answer, forgot about this one...
it happens if you have libbacktrace installed because arrow detects it and tries to enable backtrace support, the problem with that is that arrow expects the libc version of libbacktrace so obviously wont work with this version.

**Reply by revelator** [reply]
adding this to cmake flags fixes the above.
-DARROW_WITH_BACKTRACE=OFF
